TOC Meeting Agenda 04 May 2023

arsulegai opened the pull request:
1. Initial version.

arsulegai committed:
Meeting Agenda 04 May 2023

1. Initial version.

Signed-off-by: S m, Aruna <aruna.mohan@walmart.com>

tkuhrt commented on meeting-minutes/2023/2023-05-04-TOC-meeting-record.md:
```diff
@@ -0,0 +1,56 @@
+    * Peter volunteered to open [issue](https://github.com/hyperledger/transact/issues/375) and [pull request](https://github.com/hyperledger/transact/pull/376) stating that the work is being moved to libsawtooth (opened on April 4, 2023)
+    * No update on issue or pull request as of April 11, 2023
+    * Arun reached out directly to Andrea and Shawn regarding the pull request, and it has since been merged
+    * Daniela will reach out to Shawn about status and moving to EOL
```
```suggestion
    * Daniela will reach out to Shawn about status and moving to EOL
    * Shawn would like to move the project to Dormant state (see decision below)
    
* 2023 Q2 Hyperledger Sawtooth (**due 27 Apr 2023**)
```

arsulegai replied:
Added the decision line.

tkuhrt commented on meeting-minutes/2023/2023-05-04-TOC-meeting-record.md:
```diff
@@ -0,0 +1,56 @@
+    * Daniela will reach out to Shawn about status and moving to EOL
+
+# Upcoming reports
+* 2023 Q2 Hyperledger Sawtooth (due 27 Apr 2023)
```
```suggestion
```
This is now overdue. Please send a reminder to the Sawtooth community and link above.

arsulegai replied:
Done, shared the latest update.

tkuhrt commented on meeting-minutes/2023/2023-05-04-TOC-meeting-record.md:
```diff
@@ -0,0 +1,56 @@
+* [2023 TOC Project Update Calendar](https://wiki.hyperledger.org/display/TSC/2023+TOC+Project+Update+Calendar)
+
+# Discussion
+* Move Transact to Dormant.
```
```suggestion
* Decision: [Move Transact to Dormant](https://github.com/hyperledger/toc/issues/98).
```

arsulegai replied:
Updated, thanks.

arsulegai committed:
Add latest reports and update links for the discussions

1. Add missing discssion links and discord reference
   conversations.
2. Add latest reports that have come in since the time
   agenda was published.

Signed-off-by: S m, Aruna <aruna.mohan@walmart.com>

ryjones commented:
Please see #103
